Drivers/Periphs/Src/SDCard.c

```diff
@@ -0,0 +1,104 @@
+
+#include <stdio.h>
+#include <string.h>
+#include <stdarg.h> //for va_list var arg functions
+#include "fatfs.h"
+#include "SDCard.h"
+#include "main.h"
+
+FRESULT fres;   //Result after operations
```

Comment:
I think "fres" might not be the best variable name, since it is not obvious what it stands for